Assets/StreamingAssets/EOS/eos_android_config.json

```diff
@@ -26,7 +26,8 @@
     "HttpRequestIo": 0,
     "RTCIo": 0,
     "EmbeddedOverlayMainThread": 0,
-    "EmbeddedOverlayWorkerThreads": 0
+    "EmbeddedOverlayWorkerThreads": 0,
+    "ApiVersion": 3
```

Comment:
Is this API field from the SDK?

Reply:
`InitializeThreadAffinity` has _it's own_ `ApiVersion` attribute. Including it here allows us to detect instances where the object defined in the EOS SDK differs from the version for which there are stored configuration values.

A notable example of this is recently both fields `EmbeddedOverlayMainThread` and `EmbeddedOverlayWorkerThreads` were added to the object, and the `ApiVersion` was bumped from `2` to `3`.